File: src/main/webapp/scripts/Types/Evacuee.js
function Evacuee(jsonObject) {
    this.userId = jsonObject["userId"];
    this.safe = jsonObject["safe"];
    this.acknowledged = jsonObject["acknowledged"];
    this.evacId = jsonObject["evacId"];
    this.latitude = jsonObject["location"]["latitude"];
    this.longitude = jsonObject["location"]["longitude"];
    this.notification_sent = jsonObject["notificationSent"];
}

File: src/main/java/edu/unr/cse/ginac/bee/types/Evacuee.java
package edu.unr.cse.ginac.bee.types;

import java.sql.Timestamp;

public class Evacuee {
    public String userId;
    public String notification_token;
    public Coordinate location;
    public boolean safe;
    public Timestamp markedSafeAt;
    public boolean acknowledged;
    public Timestamp acknowledgedAt;
    public String evacId;
    public boolean notificationSent;
    public Timestamp notificationSentAt;
    public Timestamp locationUpdatedLastAt;

    public Evacuee() { }
}
